src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/lambda_functions/src/handler.py

```diff
@@ -0,0 +1,124 @@
+import json
+
+from destination import circleci
+from models.destination_type import DestinationType
+from models.source_type import SourceType
+from source_data_generator import aws_session_credential_source
+
+
+def handler(event, context, *, iam=None, sts=None, secretsmanager=None):
+    """
+    Invoked with the following event structure:
+    ```
+    {
+      "sources": [
+        {
+          "type": "aws_session_cred",
+          "description": "Temporary AWS Credentials to upload the release
+                          artifacts to S3 and invalidate Cloudfront",
```

Comment:
Multi-line JSON string not allowed. I know it's a comment, but it'd be nice to make this copy/pasteable